src/verify.c

```diff
@@ -606,6 +606,8 @@ int verify_main(int argc, char **argv)
 	struct manifest *official_manifest = NULL;
 	int ret;
 	int lock_fd;
+	int retries = 0;
+	int timeout = 5;
```

Comment:
I think not, and I'm a little tempted to lower the ones in update (in a different patch) to be consistent.